underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -101,6 +111,19 @@ public GCSUnderFileSystem(AlluxioURI uri) throws ServiceException {
     // TODO(chaomin): maybe add proxy support for GCS.
     mClient = new GoogleStorageService(googleCredentials);
     mBucketPrefix = PathUtils.normalizePath(Constants.HEADER_GCS + mBucketName, PATH_SEPARATOR);
+
+    mAccountOwnerId = mClient.getAccountOwner().getId();
+    // Gets the owner from user-defined static mapping from GCS account id to Alluxio user name.
+    String owner = CommonUtils.getValueFromStaticMapping(
+        Configuration.get(PropertyKey.UNDERFS_GCS_OWNER_ID_TO_USERNAME_MAPPING), mAccountOwnerId);
+    // If there is no user-defined mapping, use the display name.
+    if (owner == null) {
+      owner = mClient.getAccountOwner().getDisplayName();
+    }
+    mAccountOwner = owner == null ? "" : owner;
```

Comment:
Yeah, it makes sense to use `id` so that the user can be aware that the user-defined mapping is required. Updated, thanks.